plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTableBuilder.java

```diff
@@ -32,6 +32,7 @@
 import java.util.Set;
```

Comment:
"Fix filter and in clause parsing" is a little vague. Can you add some details in the commit message explaining what was broken?

Reply:
Updated.